internal/faustjs.org/docs/reference/WordPressBlocksViewer.mdx

```diff
@@ -0,0 +1,60 @@
+---
+slug: /reference/WordPressBlocksViewer
+title: WordPressBlocksViewer Reference
+description: Reference docs for the WordPressBlocksViewer Component in Faust.
+---
+
+`WordPressBlocksViewer` is a component used to render blocks received from WordPress. This is useful for blocks that want to render additional blocks as children as a part of their design, i.e. `CoreColumn`.
```

Comment:
You need to mention that it can be used for all content blocks as well and not the `children` only.